app/presenters/api/v2/measures/measure_presenter.rb

```diff
@@ -26,6 +18,17 @@ def vat
           vat?
         end
 
+        def duty_expression
+          Api::V2::Measures::DutyExpressionPresenter.new(self, @declarable)
+        end
+
+        def national_measurement_units
+          @national_measurement_units ||= @declarable.national_measurement_unit_set
+                                                    &.national_measurement_unit_set_units
+                                                    &.select(&:present?)
```

Comment:
I am aware you didn't write it, I think this method has the wrong name,

- if `present?` is the a method from Rails, it is superfluous, for example:

```
 [1,2,3,4,5].select(&:present?)
=> [1, 2, 3, 4, 5]
```

otherwise, it has the wrong name IMO.